runtime/tchannel_client.go

```diff
@@ -219,8 +219,14 @@ func (c *tchannelOutboundCall) finish(err error) {
 }
 
 func (c *tchannelOutboundCall) logFields() []zapcore.Field {
+	var hostPort string
+	if c.call != nil {
+		hostPort = c.call.RemotePeer().HostPort
+	} else {
+		hostPort = "unknown"
+	}
```

Comment:
nit:
```
hostPort := "unknown"
if c.call != nil {
    hostPort = c.call.RemotePeer().HostPort
}
```
prefer no `else` whenever possible

Reply:
we discussed this and decided to leave it as it is. I prefer no `else` when possible but in this case it would mean we always assign an incorrect value of "unknown" in case of `c.call != nil` which feels weird.